lib/galaxy/datatypes/binary.py

```diff
@@ -2962,6 +2962,48 @@ def display_peek(self, dataset):
             return "Binary pretext file (%s)" % (nice_size(dataset.get_size()))
 
 
+class JP2(Binary):
+    """
+    JPEG 2000 binary image format
+    >>> from galaxy.datatypes.sniff import get_test_fname
+    >>> fname = get_test_fname('test.jp2')
```

Comment:
@annefou you forget to add this test file. It needs to be in the test folder inside the lib/galaxy/datatypes folder

Reply:
oops... Of course... I was wondering how tests were done... That makes sense!!!! Thank you!